ff/src/fields/models/quadratic_extension.rs

```diff
@@ -255,27 +260,44 @@ impl<'a, P: QuadExtParameters> SquareRootField for QuadExtField<P>
 where
     P::BaseField: SquareRootField,
 {
+    /// Outputs a LegendreSymbol, which indicates whether this field element is
+    ///  1 : a quadratic residue
+    ///  0 : equal to 0
+    /// -1 : a quadratic non-residue
```

Comment:
Maybe we move this to `SqrtField` trait itself?

Reply:
Good point, I'll do that